maxim/tests/test_add_dataset_entries_comprehensive.py

```
@@ -264,15 +260,15 @@ def test_add_dataset_entries_with_file_data_attachment(self) -> None:
             self.maxim.add_dataset_entries(self.dataset_id, [entry])
 
         # Verify the upload process was initiated
-        self.assertEqual(mock_client.request.call_count, 4)
+        self.assertEqual(mock_client.request.call_count, 2)
```

Ruff (0.12.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)